common/changes/office-ui-fabric-react/textfield_font_2019-02-15-23-14.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Loading font from theme for Textfield.",
```

Comment:
```suggestion
      "comment": "TextField: adds font styles passed from theme to the root element.",
```

Reply:
I'll commit this suggestion once I finish fixing the build failures.